rpc/modules/system.go

```diff
@@ -92,21 +91,24 @@ func (sm *SystemModule) Version(r *http.Request, req *EmptyRequest, res *StringR
 
 // Health returns the information about the health of the network
 func (sm *SystemModule) Health(r *http.Request, req *EmptyRequest, res *SystemHealthResponse) error {
-	// TODO: migrate from api to network state
-	res.Health = sm.api.NetworkModule.Health()
+	// TODO: rename to not have `Get` to match API
```

Comment:
let's open an issue to address this

Reply:
this is actually not relevant anymore, I've removed it